api.js

```diff
@@ -0,0 +1,17 @@
+async function login(email, password) {
+  const response = await fetch('/api/login', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ email, password }),
+  });
+
+  if (response.ok) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Returning a boolean value from the login function might not be very informative. Consider throwing an error or returning the response object itself to provide more context about what went wrong in case of a failed request.